schemas/constructs/v1beta1/designs.json

```diff
@@ -92,59 +42,14 @@
       ]
     },
     "relationships": {
-      "type": "array",
       "description": "List of relationships between components",
+      "type": "array",
       "items": {
-        "type": "object",
-        "properties": {
-          "schemaVersion": {
-            "$ref": "../v1alpha2/relationship.json#/properties/schemaVersion"
-          },
-          "version": {
-            "$ref": "../v1alpha2/relationship.json#/properties/version"
-          },
-          "from": {
-            "type": "string",
-            "format": "uuid",
-            "description": "ID (UUID) of the source component"
-          },
-          "to": {
-            "type": "string",
-            "format": "uuid",
-            "description": "ID (UUID) of the target component"
-          },
-          "kind": {
-            "$ref": "../v1alpha2/relationship.json#/properties/kind",
-            "description": "Kind of the relationship"
-          },
-          "type": {
-            "$ref": "../v1alpha2/relationship.json#/properties/type",
-            "description": "Type of the relationship"
-          },
-          "subType": {
-            "$ref": "../v1alpha2/relationship.json#/properties/subType",
-            "description": "Subtype of the relationship (optional)"
-          },
-          "status": {
-            "type": "string",
-            "enum": [
-              "evaluating",
-              "valid",
-              "invalid"
-            ],
-            "description": "Status of the relationship"
-          }
-        },
-        "required": [
-          "from",
-          "to",
-          "kind",
-          "type",
-          "status"
-        ]
+        "$ref": "../v1alpha3/relationship.json"
       }
     }
   },
+  

```

Comment:
Relationships required? Can be nil?

A valid design can just have one component with no relationships.

Is there value in requiring relationships?